From 1311bcf790d161de8e562a6c5d7befda01264b2e Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:42:58 -0400
Subject: [ethernet] cxgb4: Adds device ID for few more Chelsio Adapters

Message-id: <1410475447-94392-40-git-send-email-svemuri@redhat.com>
Patchwork-id: 93824
O-Subject: [RHEL7.1 PATCH BZ 1124947 039/108] cxgb4: Adds device ID for few more Chelsio Adapters
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 0183aa626f467dad278050f0918a243164ae5916)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index 9b97783..11e0792 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -256,6 +256,12 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5013, 4),
  CH_DEVICE(0x5014, 4),
  CH_DEVICE(0x5015, 4),
+ CH_DEVICE(0x5080, 4),
+ CH_DEVICE(0x5081, 4),
+ CH_DEVICE(0x5082, 4),
+ CH_DEVICE(0x5083, 4),
+ CH_DEVICE(0x5084, 4),
+ CH_DEVICE(0x5085, 4),
  CH_DEVICE(0x5401, 4),
  CH_DEVICE(0x5402, 4),
  CH_DEVICE(0x5403, 4),
@@ -277,6 +283,12 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x5413, 4),
  CH_DEVICE(0x5414, 4),
  CH_DEVICE(0x5415, 4),
+ CH_DEVICE(0x5480, 4),
+ CH_DEVICE(0x5481, 4),
+ CH_DEVICE(0x5482, 4),
+ CH_DEVICE(0x5483, 4),
+ CH_DEVICE(0x5484, 4),
+ CH_DEVICE(0x5485, 4),
  { 0, }
 };
 
-- 
1.7.1